kale/embed/gripnet.py

```diff
@@ -241,3 +245,294 @@ def forward(
             z, target_edge_index, edge_type=target_edge_relations, range_list=target_edge_range, if_catout=True
         )
         return z
+
+
+class GripNetInternalModule(Module):
+    r"""
+    The internal module of a supervertex, which is composed of an internal feature layer and multiple internal
+    aggregation layers.
+
+    Args:
+        in_channels (int): the dimension of node features on this supervertex.
+        num_edge_type (int): the number of edge types on this supervertex.
+        if_start_svertex (bool): if this supervertex is a start supervertex on the supergraph.
```

Comment:
Weird bool variable name. See how we define bool variables and describe them in PyKale (search bool) or PyTorch

Reply:
You mean whether, not if here.

Reply:
Correct all bool variables named as if_xxx

Reply:
Fixed in [5c193a3](https://github.com/pykale/pykale/pull/338/commits/5c193a39c6e3d76aa9249ae5ff9546a17d6c60a2).